pkg/arvo/lib/bip/b158.hoon

```diff
@@ -157,7 +157,7 @@
   ++  to-range
     |=  [item=byts f=@ k=byts]
     ^-  @
-    (rsh [0 64] (mul f (swp 3 dat:(siphash k item))))
+    (rsh [0 64] (mul f +:(flim:sha (siphash k item))))
```

Comment:
This should probably use `+rev` directly, instead of passing through some `sha`-related helper. `+rev` is explicitly the leading-zero-safe version of `+swp`.

```suggestion
    (rsh [0 64] (mul f (rev 3 (siphash k item))))
```